test/internal/impl/laravel/5.8/handler/generator/model_generator_test.go

```diff
@@ -0,0 +1,50 @@
+package generator
```

Comment:
I thought we decided to add a genTest method with several differnt arguments and we'd iterate and match them

Reply:
Done